src/participation.ts

```diff
@@ -19,7 +19,7 @@ type ParticipationOptions = {
 class ParticipationTableElement extends HTMLTableElement {
```

Comment:
I had an idea to try of a Map, we could add 

type Candidate = Map<string,NamedNode>

Reply:
We can do this instead 
type CandidateEntry = string | NamedNode
type CanidateEntries = CandidateEntry[]